lang/th.js

```diff
@@ -20,19 +20,19 @@ export default {
 	"intl-common:actions:search": "ค้นหา", // search action (in Title Case)
 	"intl-common:actions:submit": "ส่ง", // submit action (in Title Case)
 	"intl-common:actions:update": "อัปเดต", // update action (in Title Case)
-	"intl-common:characters:ampersand": "ampersand", // short name or description of the "&" character
-	"intl-common:characters:apostrophe": "apostrophe", // short name or description of the "'" character
-	"intl-common:characters:asterisk": "asterisk", // short name or description of the "*" character
-	"intl-common:characters:backslash": "backslash", // short name or description of the "\" character
-	"intl-common:characters:colon": "colon", // short name or description of the ":" character
-	"intl-common:characters:comma": "comma", // short name or description of the "," character
-	"intl-common:characters:greaterThan": "greater-than sign", // short name or description of the ">" character
-	"intl-common:characters:lessThan": "less-than sign", // short name or description of the "<" character
-	"intl-common:characters:numberSign": "number sign", // short name or description of the "#" character
-	"intl-common:characters:percentSign": "percent sign", // short name or description of the "%" character
-	"intl-common:characters:pipe": "pipe", // short name or description of the "|" character
-	"intl-common:characters:questionMark": "question mark", // short name or description of the "?" character
-	"intl-common:characters:quotationMark": "quotation mark", // short name or description of the '"' character
+	"intl-common:characters:ampersand": "&", // short name or description of the "&" character
+	"intl-common:characters:apostrophe": "'", // short name or description of the "'" character
+	"intl-common:characters:asterisk": "*", // short name or description of the "*" character
+	"intl-common:characters:backslash": "\\", // short name or description of the "\" character
+	"intl-common:characters:colon": ":", // short name or description of the ":" character
+	"intl-common:characters:comma": ",", // short name or description of the "," character
+	"intl-common:characters:greaterThan": ">", // short name or description of the ">" character
+	"intl-common:characters:lessThan": "<", // short name or description of the "<" character
+	"intl-common:characters:numberSign": "#", // short name or description of the "#" character
+	"intl-common:characters:percentSign": "%", // short name or description of the "%" character
+	"intl-common:characters:pipe": "|", // short name or description of the "|" character
+	"intl-common:characters:questionMark": "?", // short name or description of the "?" character
+	"intl-common:characters:quotationMark": "\"-\"", // short name or description of the '"' character
```

Comment:
These are wrong (again), we want the description of the character not the character itself. Will re-raise with the localization folks.